.github/workflows/publish-stable.yml

```diff
@@ -114,6 +114,16 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  comment:
+    name: Comment on PRs
+    runs-on: buildjet-4vcpu-ubuntu-2204
+    needs: [tag, npm]
```

Comment:
to be considered if we want to run this before or after `docker` job

Reply:
Only thing I'd want to make sure of is that it can't/won't fail and stop the docker step from running.